mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -255,6 +275,9 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 		ARG 2 count
 	METHOD m_hobjxuks getBrain ()Lnet/minecraft/unmapped/C_rjqjaxef;
```

Comment:
I just realized... MobEntity has goals and LivingEntity doesn't, but why tf does LivingEntity have brains if it was meant for mob ai????? Oh god that means players can have brains thats cursed

Reply:
lmao